hack/verify-golint.sh

```diff
@@ -15,10 +15,11 @@
 # limitations under the License.
 
 set -euo pipefail
-curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin latest
+LINT_VERSION=v2.0.2
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin ${LINT_VERSION}
 if [[ -z "$(command -v golangci-lint)" ]]; then
   echo "Cannot find golangci-lint. Installing golangci-lint..."
-  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.60.1
+  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin ${LINT_VERSION}
```

Comment:
It tries to download the same lint twice, IMO one attempt would be enough...

Reply:
I refactored this a bit more in the latest commit. There was already a `make deps` target which was supposed to install it, but the old `go install` line didn't work with v2.0.2.

So I fixed the install line in Makefile, pinned it to v2.0.2, set a dependency on `deps` for targets which use `golangci-lint`, and changed `hack/verify-golint.sh` to return an error if it's not installed. Now the version is set only once and golangci-lint is installed in only one place.